src/test/java/tr/com/srdc/ontmalizer/test/XSD2OWLTest.java

```diff
@@ -107,4 +107,64 @@ public void writerTest() {
         }
     }
 
+    
+    @Test
+    public void cdaNullCheck() {
+
+        // This part converts XML schema to OWL ontology.
+        XSD2OWLMapper mapping = new XSD2OWLMapper(new File("src/test/resources/CDA/CDA.xsd"));
+        mapping.setObjectPropPrefix("");
+        mapping.setDataTypePropPrefix("");
+        mapping.convertXSD2OWL();
+
+        // This part prints the ontology to the specified file.
+        FileOutputStream ont;
+        try {
+            File f = new File("src/test/resources/output/cda-ontology.n3");
+            f.getParentFile().mkdirs();
+            String temp =    f.getParentFile().toString();
+            ont = new FileOutputStream(f);
+            if(ont != null) {//if file output stream does not be null we do the mapping 
+            	   mapping.writeOntology(ont, "N3");
+                   ont.close();
+            }
+            else 
+            	throw new Exception("File output stream doesn't have any value , you can check the file :"+ temp);
+         
+        } catch (Exception e) {
+            LOGGER.error("{}", e.getMessage());
+        }
+    }
+    
+    
+    @Test
+    public void xmlToOwlFileCheck() {
+
+    	String temp  = "src/test/resources/test/test.xsd";
+  try {
+    	if(temp.contains(".xsd")) {
```

Comment:
Nice idea, however don't just check the string since order will mater (in case the .xsd will be at the start or the middle of the file path), but check the actual file type. use assertion